docs/change_log/release-3.2.md

```diff
@@ -94,3 +94,5 @@ The following bug fixes are included in the 3.2 release:
 * Unescape backslash-escaped characters in TOC ids (#864).
 * Refactor bold and italic logic in order to solve complex nesting issues (#792).
 * Always wrap CodeHilite code in `code` tags (#862)
+* `_extend_markdown()` now correctly reports if an extension's `extend_markdown()` raises a TypeError (#939).
+* Attempting to delete a nonexistent key from the patterns registry now raises KeyError instead of TypeError (#939). 
```

Comment:
Version 3.2 has already been released. Future changes should not be included in the release notes for that version. Instead, these notes should be added to the changelog here:

https://github.com/Python-Markdown/markdown/blob/ada40c6619fd0be740af646b043937e560716c95/docs/change_log/index.md#L6-L8

Also, `TypeError` and `KeyError` are causing the [spellchecking][1] test to fail. As they are Python objects, they should be in code spans.

Finally, the [Contributing Guide][2] states:

> Use the present tense (“Add feature” not “Added feature”).
> 
> Use the imperative mood (“Move item to…” not “Moves item to…”).

Generally, the same guidelines should apply to release notes.

[1]: https://travis-ci.org/github/Python-Markdown/markdown/jobs/676115628#L312
[2]: https://python-markdown.github.io/contributing/#commit-message-style-guide